log/devlog/devlog.go

```diff
@@ -0,0 +1,118 @@
+// Package devlog provides a plain-text logger suitable for reading from the
+// console.
+//
+// For production, the team prefers JSON logging, but plain-text can be useful
+// for local development and tests.
+//
+// So while this package shouldn't be used for production, it can be enabled
+// for your development use by setting TIDEPOOL_USE_DEVLOG=1 as an environment
+// variable. It will also be enabled if POD_NAMESPACE=default (this is set via
+// the development repo when using it's kind k8s cluster).
```

Comment:
Update the comment to match the latest `TIDEPOOL_LOGGER_PACKAGE` environment variable?